server.js

```diff
@@ -33,6 +34,7 @@ app.get("/", (req, res) => res.send(`Hey Base Route!!!`));
 // Use Routes
 app.use("/api/v1/auth", auth);
 app.use("/api/v1/auth", user);
+app.use("/api/v1/auth", address);
```

Comment:
Can we have products in a separate `/api/v1/product` route please? @swetshaw

Reply:
Yeah Sure.